client/src/components/BoardUI/Card.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import styled from 'styled-components';
+import { Draggable } from 'react-beautiful-dnd';
+
+const Container = styled.div`
```

Comment:
we're using styled-components here? This is okay - but we'll have to stay consistent across the entire app. may want to check in with the team this is the direction we'd like to go

Reply:
Agreed, checked in with team and working on implementing MUI's styling solution.